invenio_rdm_records/contrib/journal/custom_fields.py

```diff
@@ -95,7 +95,7 @@ def mapping(self):
                     "description": "",
                 },
                 "pages": {
-                    "label": _("Pages"),
+                    "label": _("Page Range or Article Number"),
```

Comment:
to keep it consistent with other fields' labels:

```suggestion
                    "label": _("Page range or article number"),
```